tests/playbooks/roles/install-devstack/templates/local.conf.j2

```diff
@@ -0,0 +1,116 @@
+[[local|localrc]]
+RECLONE=False
+HOST_IP={{ local_ip_address }}
+DEST=/opt/stack
+DATA_DIR=${DEST}/data
+USE_PYTHON3=True
+LOGFILE=$DEST/logs/stack.sh.log
+VERBOSE=True
+LOG_COLOR=False
+LOGDAYS=1
+SERVICE_TIMEOUT=300
+
+DATABASE_PASSWORD=password
+ADMIN_PASSWORD=password
+SERVICE_PASSWORD=password
+SERVICE_TOKEN=password
+RABBIT_PASSWORD=password
+
+TARGET_BRANCH={{ branch }}
+
+ENABLED_SERVICES={{ enable_services | join(',') }}
+
+{% if "nova" in enable_services %}
+# Nova
+enable_plugin nova https://opendev.org/openstack/nova.git {{ branch }}
```

Comment:
Curious - what does this mean to achieve? Nova doesn't have a devstack plugin

Reply:
Thanks @gouthampacha , I'll remove these...